composer.json

```diff
@@ -139,7 +140,7 @@
         "contao-community-alliance/composer-plugin": "<3.0",
         "contao/core": "*",
         "contao/manager-plugin": "<2.0 || >=3.0",
-        "doctrine/annotations": "<1.7",
+        "doctrine/annotations": "<1.9",
```

Comment:
Should we add this conflict in the `core-bundle/composer.json` file as well?

Reply:
I’m not sure, the prefer-lowest test failed because doctrine was confused by the `@template` annotation. Don’t know if this would have an effect outside the tests too.

Reply:
https://github.com/contao/contao/runs/3035275702?check_suite_focus=true#step:7:16

Reply:
It only works because we are not testing the single bundles with `--prefer-lowest`. Otherwise it would fail there, too.

Reply:
The same applied for the existing conflict with <1.7 which was added in 74d867306620d8b070183da1a2a0ea36f86cc998
But I’m happy either way, whatever you prefer ☺️

Reply:
Let's just leave it as is.